### PR TITLE
ADEN-8168 Adding Confiant service caller

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: src/ad-services/confiant/index.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+		if (!context.get('services.confiant.enabled') || !propertyId || !mapping || !activation) {
+			utils.logger(logGroup, 'disabled');
+
+			return Promise.resolve();
+		}
+
+		utils.logger(logGroup, 'loading');
+
+		/* eslint-disable */
```
Comment (fraszczakszymon) [inline src/ad-services/confiant/index.js]: Why? :p we can defined _clrm as global

File: src/ad-services/confiant/index.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+		window._clrm = window._clrm || {};
+		window._clrm.gpt = {
+			propertyId: propertyId,
+			confiantCdn: scriptDomain,
+			sandbox: 0,
+			mapping: mapping,
+			activation: activation,
+			callback: function(blockingType, blockingId, isBlocked, wrapperId, tagId, impressionData) {
+				console.log('w00t one more bad ad nixed.', arguments);
```
Comment (fraszczakszymon) [inline src/ad-services/confiant/index.js]: Is it needed?

File: src/ad-services/confiant/index.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+	 */
+	call() {
+		const propertyId = context.get('services.confiant.propertyId');
+		const mapping = context.get('services.confiant.mapping');
+		const activation = context.get('services.confiant.activation');
+
+		if (!context.get('services.confiant.enabled') || !propertyId || !mapping || !activation) {
+			utils.logger(logGroup, 'disabled');
+
+			return Promise.resolve();
```
[the remark this answers is not in the document]
Comment (jacekbj) [reply src/ad-services/confiant/index.js]: In Krux and MoatYI we return `Promise.resolve()`.

File: src/ad-services/confiant/index.js (hunk 1 above)
Comment (dmnsobczak) [reply src/ad-services/confiant/index.js]: It's a part of code copied from Confiant ready-to-paste script tag, so I wanted to mark this as an alien code. But you are probably right :)

File: src/ad-services/confiant/index.js (hunk 2 above)
Comment (dmnsobczak) [reply src/ad-services/confiant/index.js]: It's a part of code copied from Confiant ready-to-paste script tag. Looks that we can remove it, but guess I was not brave enough :<

File: src/ad-services/confiant/index.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (dmnsobczak) [reply src/ad-services/confiant/index.js]: +1, I keeped consistency with other modules.

File: src/ad-services/confiant/index.js [hunk 4]
```diff
@@ -34,19 +34,16 @@ class Confiant {
...
 		window._clrm = window._clrm || {};
+		// eslint-disable-next-line  no-underscore-dangle
 		window._clrm.gpt = {
-			propertyId: propertyId,
+			propertyId,
 			confiantCdn: scriptDomain,
 			sandbox: 0,
-			mapping: mapping,
-			activation: activation,
-			callback: function(blockingType, blockingId, isBlocked, wrapperId, tagId, impressionData) {
```
Comment (dmnsobczak) [inline src/ad-services/confiant/index.js]: I wonder if removing this callback parameter won't break the Confiant code.

**Pull request merged**